src/translations/nbTranslations.json

```diff
@@ -612,8 +612,9 @@
       "doi_closed": "DOI avvist",
       "doi_completed": "DOI tildelt",
       "doi_pending": "Venter på DOI-tildeling",
-      "doi_requested": "Bedt om DOI",
-      "files_published": "Fil(er) publisert",
+      "files_published_one": "{{count}} fil publisert",
```

Comment:
Nødvendig med count-variabel når det er implied at det alltid skal være 1?

Reply:
Egentlig ikke nei. Men count må sendes inn som argument der den brukes da, sånn at react-i18next vet om den skal bruke teksten for `files_published_one` eller `files_published_other`, siden denne brukes sånn: `t('files_published', { count })` (uten postfiks `one`/`other`, som er noe rammeverket bruker for å håndtere pluralisering)
Så det blir egentlig det samme uansett da

Reply:
Ah, okei 👍